.github/workflows/gh-pages.yml

```diff
@@ -58,21 +62,33 @@ jobs:
           cmd: yq '.plugins[] | .i18n.languages | select( . != null ) | map(.locale) | join(",")' mkdocs.yml
 
       - name: Pull translations from Transifex
+        if: contains(fromJSON('["push", "workflow_dispatch", "schedule"]'), github.event_name) || ${{ github.event_name == 'pull_request' && github.repository == 'opengisch/QField-docs' }}
         run: |
           ./tx pull -t -l ${{ steps.get_langs.outputs.result }}
           ./tx status
         env:
           TX_TOKEN: ${{ secrets.TX_TOKEN }}
 
+      - name: Translate Mkdocs config
+        if: contains(fromJSON('["push", "workflow_dispatch", "schedule"]'), github.event_name) || ${{ github.event_name == 'pull_request' && github.repository == 'opengisch/QField-docs' }}
+        run: |
+          ./utils/mkdocs_tx.py -s en update_config
+          ./utils/mkdocs_tx_commit.sh
```

Comment:
@3nids do you have any specific reason to commit the mkdocs YAML file with the translations? 

Is it just to make the mkdocs file look nice at any point of time, or there is a technical reason?

In my understanding we can update the YAML file without commiting, then build the docs and upload them. This way we can have the `master` branch as a protected branch.

Reply:
There are several possibilities for navigation entries (dict, list, etc) and it's not super robust. Having the mkdocs committed is a warranty that it's valid and that it's correctly translated.